s3torchconnectorclient/python/tst/integration/conftest.py

```diff
@@ -120,3 +124,15 @@ def checkpoint_directory(request) -> BucketPrefixFixture:
 @pytest.fixture
 def empty_directory(request) -> BucketPrefixFixture:
     return get_test_bucket_prefix(f"{request.node.name}/empty_directory")
+
+
+@pytest.fixture
+def copy_directory(request) -> tuple[BucketPrefixFixture, str, str]:
+    fixture = get_test_bucket_prefix(f"{request.node.name}/copy_directory")
+    src_key, dst_key = "src.txt", "dst.txt"
+
+    # set up / teardown
+    fixture.add(src_key, b"Hello, World!\n")
+    fixture.remove(dst_key)
+
+    return fixture, fixture.prefix + src_key, fixture.prefix + dst_key
```

Comment:
This should probably be made into a [`dataclass`](https://docs.python.org/3/library/dataclasses.html)